build_tools/docker/nvidia/Dockerfile

```diff
@@ -45,8 +54,19 @@ COPY --from=fetch-nvidia \
   "/fetch-nvidia/${NVIDIA_COMMON_DEB}" \
   "/fetch-nvidia/${NVIDIA_GL_DEB}" \
   "/fetch-nvidia/${NVIDIA_COMPUTE_DEB}" \
+  "/fetch-nvidia/cuda-*" \
```

Comment:
Huh this second build stage isn't really helping here anymore. I can futz with it though